osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json

```diff
@@ -576,6 +700,306 @@
           "access": "RW"
         }
       ]
+    },
+    {
+      "tag": "INTERVAL_VALUES_G",
+      "description": "Hourly interval readings of M-Bus device",
+      "class-id": 7,
+      "version": 1,
+      "obis": "0.x.24.3.0.255",
+      "group": "GAS",
+      "meterTypes": ["SP","PP"],
+      "properties": {},
+      "attributes": [
+        {
+          "id": 2,
+          "description": "buffer",
+          "datatype": "array",
+          "valuetype": "DYNAMIC",
+          "value": "EMPTY",
+          "access": "R"
+        },
+        {
+          "id": 3,
+          "description": "capture objects",
+          "datatype": "array",
+          "valuetype": "FIXED_IN_PROFILE",
+          "value": "CLOCK,2|AMR_PROFILE_STATUS_HOURLY_G,2|MBUS_MASTER_VALUE,2",
+          "access": "R"
```

Comment:
Zou dit geen array moeten zijn?

Reply:
As discussed, the value of an attribute is a String. Using an array would be complicating things. Moved the splitting to the ProfileGeneric class. This class now returns a list of Capture Objects.